homeassistant/components/wled/number.py

```diff
@@ -92,7 +92,7 @@ def available(self) -> bool:
     @property
     def native_value(self) -> float | None:
         """Return the current WLED segment number value."""
-        return getattr(
+        return getattr(  # type: ignore[no-any-return]
```

Comment:
Please revert this, I have decline this PR in the past and am not happy is has been pushed through.

Please note that this is a platinum level integration that does require approval from the codeowner

Reply:
> Please revert this, I have decline this PR in the past and am not happy is has been pushed through.
> 
> Please note that this is a platinum level integration that does require approval from the codeowner

My intention was to wait for your approval. It got merged before that unfortunately.

As for the specific change. My understanding was that you didn't want to add `type: ignore` comments for things related to the untyped `backoff` decorator. `getattr` however will always return `Any`. So if `type: ignore` is out of the question the only alternatives are `cast`, no strict typing, or a rewrite using a custom callable in the `entity_description`.

If you still want me to revert it, I'll do it. Just let me know.

Reply:
Yes, please revert it as per above.

Reply:
Yes, please revert it. The use of `getattr` should be refactored out of it first.